cpp/src/Ice/ConnectionI.h

```diff
@@ -204,7 +204,7 @@ namespace Ice
 
         void exception(std::exception_ptr);
 
-        void dispatch(
+        void upCall(
```

Comment:
Updated the comment.

> when one or more outgoing requests have been sent (I don't understand how it can be more than one)

See https://github.com/zeroc-ice/ice/blob/9ae478cdcec28dba13cbc138205a771f585d8b81/cpp/src/Ice/ConnectionI.cpp#L2841 which is called from the message method implementation. It can sent multiple messages. I've renamed this method `sendNextMessages`.

> Is it actually possible to process more than one kind of event in the same call to this function? Like process a dispatch and an incoming response in a single call? If not, why combine them all into a single function?

Yes see the comment. We can probably simplify this by using one or two `std::function` to encapsulate the upcall.